docker/build-tools/Dockerfile

```diff
@@ -404,10 +407,14 @@ RUN gem install --no-wrappers --no-document html-proofer -v ${HTMLPROOFER_VERSIO
 RUN gem install --no-wrappers --no-document awesome_bot -v ${AWESOMEBOT_VERSION}
 RUN gem install --no-wrappers --no-document licensee -v ${LICENSEE_VERSION}
 # hadolint ignore=DL3003,DL3028
-RUN git clone https://github.com/jordansissel/fpm && \
-    cd fpm && \
-    git reset --hard ${FPM_VERSION} && \
-    make install
+RUN mkdir -p /tmp/fpm && \
+    cd /tmp/fpm && \
+    git init && \ 
+    git remote add origin https://github.com/jordansissel/fpm && \
+    git fetch --depth 1 origin ${FPM_VERSION} && \
+    git checkout FETCH_HEAD && \
+    make install && \
+    rm -rf /tmp/*
```

Comment:
Should this be tmp/fpm?

Reply:
Correct, we don't need nuclear option here. I adjusted it to be similar to `test-infra` install in a38efe3.